tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/ModelXmlParserGeneratorMojo.java

```diff
@@ -223,7 +222,6 @@ private JavaClass generateParser(List<Class<?>> model, ClassLoader classLoader)
         parser.addImport(List.class);
         parser.addImport(ArrayList.class);
         parser.addAnnotation(SuppressWarnings.class).setLiteralValue("\"unused\"");
-        parser.addAnnotation(Generated.class).setLiteralValue("\"" + getClass().getName() + "\"");
```

Comment:
Can we add a javadoc like the others do with
```
/**
 * Generated by camel build tools - do NOT edit this file!
 */
```

Reply:
Yes we can. Let me try that.